docs/overview/excel.md

```diff
@@ -54,9 +54,11 @@ The Action Recorder records actions you take in Excel and saves them as a script
 
 All scripts recorded with the Action Recorder can be edited through the Code Editor. This lets you tweak and customize the script to better suit your exact needs. You can also add logic and functionality that is not directly accessible through the Excel UI, such as conditional statements (if/else) and loops.
 
-One easy way to start learning the capabilities of Office Scripts is to record scripts in Excel on the web and view the resulting code. Another option is to follow our [tutorials](../tutorials/excel-tutorial.md) to learn in a more guided and structured way.
+> [!TIP]
+> The Action Recorder has a **Copy as code** button to actions into script code without saving the entire script.
+> :::image type="content" source="../images/action-recorder-copy-code.png" alt-text="The Action Recorder task pane with the 'Copy as code' button highlighted.":::
```

Comment:
This image displays on the same line as the text in the script, creating a strange visual experience. Can we force a line break in the !TIP?

Reply:
It's unfortunate that displays correctly in the VSCode preview. Good to know there's a difference!